mappings/net/minecraft/item/BannerPatternItem.mapping

```diff
@@ -1,5 +1,12 @@
 CLASS aup net/minecraft/item/BannerPatternItem
+	FIELD a patterns Lbnk;
```

Comment:
Maybe just `pattern`, as the getter is just `getPattern()`?

Reply:
Definitely. It's not a collection. Whoops!